ui/src/publication/card/publication-card.tsx

```diff
@@ -33,12 +33,18 @@ type PublishListProps = {
     branchType: LayoutBranchType;
 };
 
+const RATKO_SUPPORT_EMAIL = 'vayla.asiakkaat.fi@cgi.com';
+export const GEOVIITE_SUPPORT_EMAIL = 'geoviite.support@solita.fi';
```

Comment:
Ei tämäkään ideaalisin paikka näille ole, mutta tällä mennään alustavasti (näitä tarvitaan vain julkaisujen yhteydessä)

Reply:
Joo. Oikeammin haluaisin itse nähdä nämä elämässä jossain parameter storessa / secrets managerissa, mutta se vaatisi jonkinlaisen logiikan näiden muiluttamiseksi sieltä ja bäkkäriendpointin niiden hakemiseen (toki voisivat tulla esim. `EnvironmentInfo`:n tai `UserInfo`:n mukana)

Reply:
Tai suoraan bäkkäriltä käännösfilun mukana, jos se ei olisi täysin staattinen (tai dynaamisesti muodostetuille olisi oma käännösfilunsa) 🤔

Reply:
Joo nämä olisi kyllä hyvä raapia ulos koodista. Tuon voisi tiketöidä ja priorisoida työn alle.

Reply:
Vaikutti siltä että tästä ei oltu vielä tehty tikettiä, joten tein sellaisen nyt: https://extranet.vayla.fi/jira/browse/GVT-2838